.github/ISSUE_TEMPLATE/bug-issue.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+labels: 'bug'
+title: '[BUG]'
+
+---
+
+## Bug
+
+### Environment information
+
+* Delta Lake version:
+* Spark version:
+* Scala version:
```

Comment:
Let's move this section below "Describe the problem" and above "Willingness to contribute"

Reply:
Ok.